client/src/templates/Challenges/components/fill-in-the-blanks.tsx

```diff
@@ -60,7 +69,7 @@ function FillInTheBlanks({
                       onChange={handleInputChange}
                       data-index={node.value}
                       size={blankAnswers[value].length}
-                      aria-label={t('learn.blank')}
+                      aria-label={getInputLabel(value)}
```

Comment:
```suggestion
                      aria-label={t('learn.fill-in-the-blank.blank')}
                      autoComplete='off'
                      {...(answersCorrect[value] === false ? { 'aria-invalid': 'true' } : {})}
```

The accessible name on the input should not change, it should remain "blank". If the answer is wrong then you can add `aria-invalid="true"`. Also, highly recommend we turn the `autocomplete` attribute off because we don't want the browser to attempt to fill these in and it will reduce the verbosity for screen reader users.

For correct answers, I highly recommend you remove the input element and just make the correct answer a part of the sentence. I did this by adding the following `if` block before this one:

```
                if (type === 'blank' && answersCorrect[value] === true)
                  return (
                    <span>{blankAnswers[value]}</span>
                  );
```
You could also format this to make it stand out as correct, such as making it bold and green. But don't underline it so it isn't confused with an active input.